src/projections/updateProjection.ts

```diff
@@ -8,10 +8,10 @@ import { debug, convertToCommandError } from "../utils";
 
 export interface UpdateProjectionOptions extends BaseOptions {
   /**
-   * Enables tracking emitted streams.
+   * Enables emitting events from the projection.
```

Comment:
**CONCERN:** I'm not sure if that's in the scope of this PR, but `Enabled` parameter is missing defining if projection should be enabled by default.

Reply:
Hm, after rereading the RFC, it might be fine. RFC states at first that there should be such a parameter:
![obraz](https://user-images.githubusercontent.com/5562528/142173028-46f3e40f-4120-414a-8139-695057c384ff.png)
But then it states that it should be removed:
> Remove the Enabled parameter from CreateProjection. This will always be false, and the projection must be enabled as a separate step with the Enable operation on the proto.
@hayley-jean thoughts?